WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -615,6 +621,10 @@ private extension ShippingLabelAddressFormViewController {
                                                             comment: "Error in finding the address in the Shipping Label Address Validation in Apple Maps")
         static let phoneNumberErrorNotice = NSLocalizedString("The phone number is not valid or you can't call the customer from this device.",
             comment: "Error in calling the phone number of the customer in the Shipping Label Address Validation")
+        static let addressValidationErrorNotice = NSLocalizedString("Certain required fields needs attention.",
```

Comment:
We can also rename this variable to clarify the two notices:
```suggestion
        static let addressLocalValidationErrorNotice = NSLocalizedString("Certain required fields need attention.",
```